webui/react/.eslintrc.js

```diff
@@ -50,10 +51,18 @@ module.exports = {
     'no-multi-spaces': [ 'error', { ignoreEOLComments: true } ],
     'no-multiple-empty-lines': [ 'error', { max: 1, maxBOF: 0, maxEOF: 0 } ],
     'no-trailing-spaces': [ 'error', {} ],
+    'object-curly-newline': [ 'error', {
+      ExportDeclaration: { consistent: true },
```

Comment:
changed this to `consistent` instead of `multiline` for import and export so we can still support something like this:
```
import {
  Checkpoint, CheckpointDetail, CheckpointState, ExperimentDetails,
  RunState, Step, TrialDetails, ValidationMetrics,
} from 'types';
```
otherwise each of those imports will be on its own line